deployer/src/runtime_manager.rs

```diff
@@ -168,6 +175,12 @@ impl RuntimeManager {
         Ok(runtime_client)
     }
 
+    pub fn kill_process(&mut self, id: Uuid) {
+        if let Some((mut process, _)) = self.runtimes.lock().unwrap().remove(&id) {
+            let _ = process.start_kill();
```

Comment:
Two questions here:
- can we also handle the error here?
- is there a way to ensure that the child is sending `wait` call to avoid creating a zombie process?

Reply:
> can we also handle the error here?

Yep, handling should be ok. I thought we didn't necessarily need it because there is also the aspect of `kill_on_drop()`, which will happen when the process is dropped, so we're somewhat confident the process will be killed in the end. I am doing this here just as an extra beside the log that will let us know the process should be killed.

> is there a way to ensure that the child is sending `wait` call to avoid creating a zombie process?

not sure what you mean by having the child send a `wait` call. Do you mean calling `wait` on the child before exiting the parent process (meaning the deployer service)? If yes, I think that if the deployer exits gracefully, the processes are killed because they have `kill_on_drop` set to true, so no need for waiting. In case of a non-graceful exit I am not sure we can do much if the runtime stays alive as a zombie process, but either way, the project is unusable at that point, and the runtime might still do some background work as a rogue process.